### PR TITLE
[NETBEANS-1074] Module Review javascript2.knockout

### PR DESCRIPTION
**Pull request by sdedic**
The model files do not seem to support comments, do not contain any license; rat exclusion added. Filed [NETBEANS-1155](https://issues.apache.org/jira/browse/NETBEANS-1155)

### CODE REVIEW

**Comment by matthiasblaesing** [comment]
Instead of adding a global rat exclude, other javascript modules were modified to contain a `licenseinfo.xml` file. This approach has the benefit, that the license situation is clear and a list of files can be created from that. See here:

https://github.com/apache/incubator-netbeans/pull/709/files

@junichi11 determined, that neighter .bowerrc, nor bower.json supported comments and so they can't hold that info, but with the info in the created `licenseinfo.xml`, a third party reviewer, can:

a) verify the license (see license/ref)
b) see the reason (comment/type)

when creating a rat report, files covered by licenseinfo.xml are excluded from the report and a summary is created as `rat-appendix.txt`, that gives a details report about these files.

File: javascript2.knockout/licenseinfo.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    specific language governing permissions and limitations
+    under the License.
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-2.2.1.model</file>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-3.0.0.model</file>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-3.2.0.model</file>
+    </fileset>
+    <license ref="Apache-2.0-ASF" />
```
Comment (matthiasblaesing) [inline javascript2.knockout/licenseinfo.xml]: license + comment need to be children of fileset

Comment (sdedic) [reply javascript2.knockout/licenseinfo.xml]: eh, silly copy/paste typo. Thanks for the hint.

**Comment by geertjanw** [comment]
Great. Squash and merging this.

**Pull request merged**